exporter/opentelemetry-exporter-otlp/src/opentelemetry/exporter/otlp/exporter.py

```diff
@@ -123,23 +124,36 @@ class OTLPExporterMixin(
         endpoint: OpenTelemetry Collector receiver endpoint
         credentials: ChannelCredentials object for server authentication
         metadata: Metadata to send when exporting
+        compression: Compression algorithm to be used in channel
     """
 
     def __init__(
         self,
         endpoint: str = "localhost:55680",
         credentials: ChannelCredentials = None,
         metadata: Optional[Tuple[Any]] = None,
+        compression: str = None,
```

Comment:
Can we use something more restrictive than raw string input here? Maybe python string literal types or your own enum.

You might also seek clarification if `Compression.Deflate` is allowed, in which case you can just use the `Compression` enum directly (I'm not sure why it wouldn't be allowed)
```suggestion
        compression: Optional[str] = None,
```

Reply:
I think in the specs (https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/protocol/exporter.md#opentelemetry-protocol-exporter) under supported compression it only lists Gzip and not Deflate